analyses/molecular-subtyping-MB/05-subtype-mb-shh.R

```diff
@@ -0,0 +1,275 @@
+# Author: Ryan Corbett & Jo Lynne Rokita
+# Function: Script to subtype MB SHH tumors 
+
+# Load packages
+suppressPackageStartupMessages({
+  library(tidyverse)
+  library(data.table)
+})
+
+# Set directory paths
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Set file paths
+data_dir <- file.path(root_dir, "data")
+analysis_dir <- file.path(root_dir, "analyses", "molecular-subtyping-MB")
+results_dir <- file.path(analysis_dir, "results")
+input_dir <- file.path(analysis_dir, "input")
+plots_dir <- file.path(analysis_dir, "plots")
+
+# set file paths
+hist_file <- file.path(data_dir, "histologies.tsv")
```

Comment:
```suggestion
hist_file <- file.path(data_dir, "histologies-base.tsv")
```
this needs to use histologies-base since the subtyping will occur before the histologies file is generated.